packages/core/package.json

```diff
@@ -2017,7 +2022,7 @@
                 "command": "aws.accessanalyzer.iamPolicyChecks",
                 "title": "%AWS.command.accessanalyzer.iamPolicyChecks%",
                 "category": "%AWS.title%",
-                "enablement": "aws.iamPolicyChecks.enabled"
+                "enablement": "config.aws.experiments.iamPolicyChecks"
```

Comment:
Do we plan to create another PR to enable this by default before the launch and release that version through the toolkits pipeline?

Reply:
I am assuming that is the process, but this will now make it available through Experiments, which is a checkbox in the settings.